### PR TITLE
GEODE-10280: add Status Message to Status Server Command

### PR DESCRIPTION
**Pull request by mivanac**
### For all changes:
- [*] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [*] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [*] Is your initial contribution a single, squashed commit?

- [*] Does `gradlew build` run cleanly?

- [x] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### CODE REVIEW

File: geode-core/src/test/java/org/apache/geode/distributed/ServerStateTest.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    map.put(AbstractLauncher.ServiceState.JSON_TIMESTAMP, timestampTime);
+    map.put(AbstractLauncher.ServiceState.JSON_UPTIME, uptime);
+    map.put(AbstractLauncher.ServiceState.JSON_WORKINGDIRECTORY, workingDirectory);
+
+    String status = null;
+    try {
+      status = new ObjectMapper().writeValueAsString(map);
+    } catch (JsonProcessingException e) {
+      e.printStackTrace();
+    }
```
Comment (kirklund) [inline geode-core/src/test/java/org/apache/geode/distributed/ServerStateTest.java:203]: Please add `throws JsonProcessingException` to the test method and remove the try-catch block.

**Review by kirklund — CHANGES_REQUESTED** [review]
Seems like a good idea overall.

File: geode-core/src/test/java/org/apache/geode/distributed/ServerStateTest.java (hunk 1 above)
Comment (mivanac) [reply geode-core/src/test/java/org/apache/geode/distributed/ServerStateTest.java]: Thanks for comments. Updated.

File: geode-core/src/test/java/org/apache/geode/distributed/ServerStateTest.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  public void fromJsonWithEmptyStringThrowsIllegalArgumentException() {
+    // given: empty string
+    String emptyString = "";
+
+    // when: passed to fromJson
+    Throwable thrown = catchThrowable(() -> fromJson(emptyString));
+
+    assertThat(thrown).isInstanceOf(IllegalArgumentException.class)
+        .hasCauseInstanceOf(NullPointerException.class);
+    assertThat(thrown.getCause()).isInstanceOf(NullPointerException.class).hasNoCause();
```
Comment (kirklund) [inline geode-core/src/test/java/org/apache/geode/distributed/ServerStateTest.java:82]: AssertJ added in some more methods like `hasRootCauseInstanceOf` that help a lot in these cases. I'm coding this off of memory but I think it ends up looking like:
```
assertThat(thrown)
  .isInstanceOf(IllegalArgumentException.class)
  .hasRootCauseInstanceOf(NullPointerException.class);
```
You probably wouldn't need any sort of assertion about there not being any intermediate causes.

**Review by kirklund — APPROVED** [review]

**Pull request merged**